go/interpreter/lfvm/ct_test.go

```diff
@@ -56,6 +58,98 @@ func TestCtAdapter_Interface(t *testing.T) {
 	var _ ct.Evm = &ctAdapter{}
 }
 
+func TestCTAdapter_DoesNotAddDuplicatedCodeToPCMap(t *testing.T) {
+	s := st.NewState(st.NewCode([]byte{
+		byte(vm.STOP),
+	}))
+	c := NewConformanceTestingTarget()
+
+	executeStepAndChecks := func() {
+		s.Status = st.Running
+		_, err := c.StepN(s, 1)
+		if err != nil {
+			t.Fatalf("unexpected conversion error: %v", err)
+		}
+		if c.(*ctAdapter).pcMapCache.Len() != 1 {
+			t.Fatalf("unexpected pc map size, wanted 2, got %d", c.(*ctAdapter).pcMapCache.Len())
+		}
+	}
+
+	executeStepAndChecks()
+	executeStepAndChecks()
```

Comment:
nit: how about using a loop?
```suggestion
	for i := 0; i<2; i++ {
		s.Status = st.Running
		_, err := c.StepN(s, 1)
		if err != nil {
			t.Fatalf("unexpected conversion error: %v", err)
		}
		if c.(*ctAdapter).pcMapCache.Len() != 1 {
			t.Fatalf("unexpected pc map size, wanted 2, got %d", c.(*ctAdapter).pcMapCache.Len())
		}
	}
```